lib/ansible/modules/messaging/rabbitmq_queue.py

```diff
@@ -99,6 +99,14 @@
             - Original routing key will be used if unset
         required: false
         default: None
+    max_priority:
+        description:
+            - Maximum number of priority levels for the queue to support.
+            - If not set, the queue will not support message priorities.
+            - Larger numbers indicate higher priority.
+        required: false
```

Comment:
Parameters with default values could be omitted (default value for `required` is `false` and default value for `default` is `null`, `required` and `default` could be removed). As other parameters specify `required` and `default`, you might want to keep them in order to preserve consistency.